components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -76,40 +76,46 @@
               *ngIf="(updateServerForm.get('ip_address').hasError('required') || updateServerForm.get('ip_address').hasError('pattern')) && updateServerForm.get('ip_address').dirty">
               IP Address is required.
             </chef-error>
-          </chef-form-field>  
+          </chef-form-field>
           <div id="button-bar">
             <chef-button [disabled]="isLoading || !updateServerForm.valid || !updateServerForm.dirty"
-              primary inline (click)="saveServer()">                
-            <chef-loading-spinner *ngIf="saveInProgress"></chef-loading-spinner>
-            <span *ngIf="saveInProgress">Saving...</span>
-            <span *ngIf="!saveInProgress">Save</span></chef-button>
+            primary inline (click)="saveServer()">
+              <chef-loading-spinner *ngIf="saveInProgress"></chef-loading-spinner>
+              <span *ngIf="saveInProgress">Saving...</span>
+              <span *ngIf="!saveInProgress">Save</span></chef-button>
             <span id="saved-note" *ngIf="saveSuccessful && !updateServerForm.dirty">All changes saved.</span>
           </div>
         </form>
       </section>
       <section class="page-body" *ngIf="tabValue === 'orgs'">
-        <ng-container>
+        <chef-loading-spinner *ngIf="orgsListLoading" size="50"></chef-loading-spinner>
+        <ng-container *ngIf="!orgsListLoading">
           <chef-toolbar>
-            <chef-button id="create-button" primary (click)="openCreateModal('create')">Add Org</chef-button>
+            <app-authorized [allOf]="['/api/v0/infra/servers', 'post']">
+              <chef-button id="create-button" primary (click)="openCreateModal('create')">Add Chef Organization
+              </chef-button>
+            </app-authorized>
           </chef-toolbar>
           <chef-table>
             <chef-thead>
               <chef-tr>
-                <chef-th>Org Name</chef-th>
+                <chef-th>Name</chef-th>
                 <chef-th>Admin</chef-th>
                 <chef-th class="three-dot-column"></chef-th>
               </chef-tr>
             </chef-thead>
             <chef-tbody>
               <chef-tr *ngFor="let org of orgs">
                 <chef-td>
-                  <a [routerLink]="['/infrastructure','chef-servers', server?.id, 'org', org.id]">{{ org.name }}</a>
+                  <a [routerLink]="['/infrastructure','chef-servers', server?.id, 'organizations', org.id]">{{ org.name }}</a>
                 </chef-td>
                 <chef-td>{{ org.admin_user }}</chef-td>
                 <chef-td class="three-dot-column">
-                  <mat-select panelClass="chef-control-menu" id="menu-{{org.id}}">
-                    <mat-option data-cy="delete" (onSelectionChange)="startOrgDelete($event, org)">Delete Org</mat-option>
-                  </mat-select>
+                  <app-authorized [allOf]="['/api/v0/infra/servers', 'post']">
```

Comment:
This technically needs the same change I mentioned above. BUT: we do not yet have an efficient way to do this for every entry in a table. So suggest leave it as a reminder but comment it out, like we have done here:
https://github.com/chef/automate/blob/72325ab808f09179be86c193efd2b817d19735a4/components/automate-ui/src/app/modules/policy/list/policy-list.component.html#L29-L33

So needs to be updated as shown:
```
<!-- <app-authorized [allOf]="['/api/v0/infra/servers/{server_id}/orgs/{id}', 'delete', [server?.id, org.id]]">
```